.travis.yml

```diff
@@ -19,5 +22,14 @@ before_cache:
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
 
+# For a PR TRAVIS_BRANCH is set to the target branch which is master.
+# We need to make sure it's not a PR when releasing.
 script:
-   - sbt ++$TRAVIS_SCALA_VERSION ciBuild
+  - if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ];
```

Comment:
To be honest, when a script in`.travis.yml` exceeds a one-liner I get tempted to extract it into a separate script e.g. in this case called `ciBuild.sh`.